addons/captives/config.cpp

```diff
@@ -6,7 +6,7 @@ class CfgPatches {
         weapons[] = {"ACE_CableTie"};
         requiredVersion = REQUIRED_VERSION;
         requiredAddons[] = {"ACE_Interaction"};
-        author[] = {"commy2", "KoffeinFlummi"};
+        author[] = {"commy2", "KoffeinFlummi", "GieNkoV"};
```

Comment:
I don't think adding stringtables is quite enough to list yourself as author of the module. @PabstMirror rewritted 3/4 of this module for ACE and it's not even listed here! (btw, he should be).

The same is true for most modules, which at this point have been extensively rewritten/ported/ammended by core developers not listed as authors. 

We may want to change to something like `author[] = {"ACE Team"};` at some point